### ISSUE TITLE
how use not only(softmax) pb for tensorflow.

### ISSUE THREAD

**Issue by pighead1016** [OAID/Tengine#130]
I found in the load_graph.use reshape op to commit the last node.
if I use other pb, like softmax is not the last node .
i change it ,create grph is ok.
but TF_SessionRun faild,
could you give some other regresion (not classification) example for the pb file of tensorflow .

like ssd or openpose

**Comment by pighead1016** [comment]
my model is here
链接: https://pan.baidu.com/s/1qD-hI15XYBPWp2KuOQC8Ew 提取码: rcbs 
How can i use this model with tengine?
